consensus/src/chained_bft/persistent_storage.rs

```diff
@@ -67,7 +70,7 @@ impl<T: Payload> RecoveryData<T> {
         mut quorum_certs: Vec<QuorumCert>,
         storage_ledger: &LedgerInfo,
         highest_timeout_certificate: Option<TimeoutCertificate>,
-        validators: Arc<ValidatorVerifier>,
+        validator_keys: ValidatorSet,
```

Comment:
Have you chatted with the network folks? I was under the impression that they might be decoupling network config from ValidatorSet?